### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by rainley123** [wizyoung/YOLOv3_TensorFlow#14]
Hello, How to understand the following sentence :
First stage: Restore darknet53_body part weights from COCO checkpoints, train the yolov3_head with big learning rate like 1e-3 until the loss reaches to a low level, like less than 1.
Second stage: Restore the weights from the first stage, then train the whole model with small learning rate like 1e-4 or smaller. At this stage remember to restore the optimizer parameters if you use optimizers like adam.
